ledger-core/v2/Makefile

```diff
@@ -51,6 +51,7 @@ clean: ## run all cleanup tasks
 	go clean $(ALL_PACKAGES)
 	rm -f $(COVERPROFILE)
 	rm -rf $(BIN_DIR)
+	rm -rf $(ARTIFACTS_DIR)
```

Comment:
I can't find using `make clean` in github actions.

Reply:
It's used in `launchnet.sh`

Reply:
In `rebuild_binaries`? but func is unused :(

Reply:
Hmm, yes, really. But nevertheless, I use it all the time manually :)

Reply:
Maybe it's time to add it in `test_func` step or in `launchnet` :)

Reply:
To run tests on a clean environment I run `make` (there is `clean` target in it) and then `make test_func`